R/g_spaghettiplot.R

```diff
@@ -182,6 +184,7 @@ g_spaghettiplot <- function(data,
                             ylim = c(NA, NA),
                             alpha = 1.0,
                             facet_ncol = 2,
+                            facet_scales = "fixed",
```

Comment:
add `match.arg` for allowed values

Reply:
yup! ab67cd6442270ac5f62fee34c89cc7e6614757b3

Reply:
this is checked anyway in ggplot2::facet_wrap

> scales <- arg_match0(scales %||% "fixed", c("fixed", "free_x", "free_y", "free"))

Reply:
And will be checked again in the module, I suppose :)
When you add the argument and an input to the module, remember to adjust the `InputValidator`.